src/main/java/com/google/sps/servlets/utils/Statistics.java

```diff
@@ -12,14 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.sps.servlets;
+package com.google.sps.servlets.utils;
```

Comment:
Is this used anywhere?

Reply:
Cis I am kind of confused with this comments. Why is this not used? We decided to change the statistics comments into a sub package under servlets since it is just a value & computation class.

Reply:
Ah for some reason I misread it as an import oops :) You are all good!